### PR TITLE
Drupal 10 compatibility

### PR DESCRIPTION
**Pull request by albeortev**
Issue #104 

Updating packages and dependencies to make the project compatible with Drupal 10.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,14 +33,14 @@
...
         "drupal-composer/drupal-security-advisories": "8.x-dev",
         "drupal/admin_toolbar": "^3.3",
-        "drupal/components": "^2.4",
+        "drupal/components": "^3.0@beta",
         "drupal/config_split": "^2.0@RC",
-        "drupal/core-composer-scaffold": "^9",
-        "drupal/core-recommended": "^9.1",
+        "drupal/core-composer-scaffold": "^10.0",
+        "drupal/core-recommended": "^10.0",
+        "drupal/drupal-extension": "^5.0@alpha",
```
Comment (omarlopesino) [inline composer.json]: - Drupal extension is  a dev library and it is being not required as require-dev
- Drupal extension is not required by required by the boilerplate but by nuvoleweb/drupal-behat which is already required by metadrop/behat-contexts, we should not require it as we rely on metadrop/behat-contexts to require the complete stack.

```suggestion
```

Comment (albeortev) [reply composer.json]: Thanks for the review, I remove the requirement of the package, it has been added unintentionally when testing the local functionality.

**Comment by omarlopesino** [comment]
@albeorte96 any advances on this?

**Comment by albeortev** [comment]
> @albeorte96 any advances on this?

Some related pending changes have been adjusted.
It is ready for testing

**Comment by omarlopesino** [comment]
@albeorte96 then please mark it as a ready for review, thanks!

**Comment by juanjol** [comment]
Tested and works great. Changes looks good to me, maybe this could be the start of Boilerplate 3.0?

**Review by omarlopesino — APPROVED** [review]

**Comment by juanjol** [comment]
Now that it is approved, should it be merged into a new branch or release? Maybe 3.0-rc, or beta?

**Pull request merged**